pkg/controllers/common_consts.go

```diff
@@ -32,4 +34,13 @@ const (
 
 	// InfrastructureResourceName is the name of the cluster global infrastructure resource.
 	InfrastructureResourceName = "cluster"
+
+	// SynchronisedCondition is used to denote when a MAPI or CAPI resource is
+	// synchronised. This condition should only be true when a synchronisation
+	// controller has successfully synchronised a non-authoritative resource.
+	SynchronisedCondition machinev1beta1.ConditionType = "Synchronised"
+
+	// ReasonResourceSynchronised denotes that the resource is synchronised
+	// successfully.
+	ReasonResourceSynchronised = "ResourceSynchronised"
```

Comment:
Now I hate to say this, as a brit, but, is this the spelling we should be using for this, given it is user facing? I believe the EP and existing API field use the US spelling